<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APTDemodSettingsDialog</class>
 <widget class="QDialog" name="APTDemodSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>212</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>APT Demodulator Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="5" column="0">
       <widget class="QLabel" name="autoSavePathLabel">
        <property name="text">
         <string>Path to save image</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="autoSavePath">
          <property name="toolTip">
           <string>Path to save images to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="autoSavePathBrowse">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../sdrgui/resources/res.qrc">
            <normaloff>:/load.png</normaloff>:/load.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="minScanlinesLabel">
        <property name="text">
         <string>Minimum scanlines</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="minScanlines">
        <property name="toolTip">
         <string>Enter the minimum number of scanlines in an image (after cropping) for it to be automatically saved</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="satelliteLabel">
        <property name="text">
         <string>Satellite</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="satellite">
        <property name="toolTip">
         <string>Select which satellite this channel will be used for</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NOAA 15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NOAA 18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NOAA 19</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="satelliteTrackerControl">
        <property name="toolTip">
         <string>Check to enable control by Satellite Tracker feature</string>
        </property>
        <property name="text">
         <string>Enable Satellite Tracker control</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="autoSave">
        <property name="toolTip">
         <string>Check to automatically save images when acquisition is stopped or LOS</string>
        </property>
        <property name="text">
         <string>Auto save image</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>satelliteTrackerControl</tabstop>
  <tabstop>satellite</tabstop>
  <tabstop>autoSave</tabstop>
  <tabstop>autoSavePath</tabstop>
  <tabstop>autoSavePathBrowse</tabstop>
  <tabstop>minScanlines</tabstop>
 </tabstops>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>APTDemodSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>APTDemodSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
